src/setupTests.js

```diff
@@ -1,17 +1,14 @@
 /* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This override is necessary?

Reply:
It is kind of necessary, as it suggest to have `deep-freeze`, `@testing-library/react` and `redux-mock-store` to be listed in the project dependencies instead of the dev dependencies, as this libraries are for testing purposes I think they should be listed in the dev dependencies.